### DISCUSSION TITLE
How to jump to next page in API ?

### DISCUSSION THREAD

**Discussion by piotrekp2fieldly** [ddnexus/pagy#507] [Q&A] [answered]
I want to "change" the page in API response and sending params like `api/mobile/v5/work_items/?page=1&per_page=10` , I set inside controller something like this `page, recs = pagy(work_items, items: params[:per_page], page: params[:page])` , but I got the error `Pagy::OverflowError (expected :page in 1..1; got 2)` , What I have been doing wrongly ? 🤔

**Comment by ddnexus** [comment, marked as answer]
In a normal environment you should not need to pass the `page` variable.

Besides, in order to dynamically change the `items` variable, you could use the items extra which is tested right for that scope.
